docs/release_notes.rst

```diff
@@ -11,6 +11,10 @@ Major Updates
 
 * Support for Python 2.6 has been dropped (#206, #217).
 
+Bug Fixes
+
+* Made parser more robust to bad source files (#168)
```

Comment:
Include the pull request as well. i.e., this should be:

    * Made parser more robust to bad source files (#168, #214).